libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/NavigatorMapper.kt

```diff
@@ -294,7 +294,7 @@ private fun RouteInfo?.toRouteInitInfo(): RouteInitInfo? {
     }
 }
 
-private fun List<com.mapbox.navigator.UpcomingRouteAlert>.toUpcomingRoadObjects():
+internal fun List<com.mapbox.navigator.UpcomingRouteAlert>.toUpcomingRoadObjects():
```

Comment:
This function should be good to be removed, right? Same as the `SUPPORTED_ROAD_OBJECTS` that's also been moved to `base` module.